SecurityService.IntegrationTests/Common/TestingContext.cs

```diff
@@ -16,6 +16,8 @@ public class TestingContext
 
         public List<String> ApiResources;
 
+        public List<String> IdentityResources;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/stuart_ferguson1/SecurityService/pullRequest?prid=5481101)